src/main/java/digital/slovensko/autogram/core/eforms/EFormResourcesBuilder.java

```diff
@@ -22,7 +22,7 @@ public static EFormResources build(DSSDocument document, String fsFormId, String
         if ((isXML(documentMimeType) || isXDC(documentMimeType)) && (eformResources == null))
             return buildFromDocument(document, fsFormId, propertiesCanonicalization, xsdIdentifier, xsltParams);
 
-        return null;
+        return eformResources;
```

Comment:
Toto sa tu preco zmenilo?

Reply:
Toto bol bug, ktorý som tu rovno fixol. Ak boli schémy embednuté v XDC, v prvom ife sa vybrali, ale nikdy sa nepoužili. (Pri FS veciach sa to menilo a trochu pokazilo)